samples/cellular/nrf_cloud_multi_service/Kconfig

```diff
@@ -220,10 +221,96 @@ config AT_CMD_REQUEST_RESPONSE_BUFFER_LENGTH
 	  error code sent in its place (-NRF_E2BIG). Cannot be less than 40 bytes.
 	default 200
 
+if NRF_CLOUD_COAP
+
+menuconfig NRF_CLOUD_COAP_FOTA
+	bool "Enable FOTA service when using CoAP to connect to nRF Cloud"
+	select FOTA_DOWNLOAD
+	select FOTA_DOWNLOAD_PROGRESS_EVT
+	select IMG_ERASE_PROGRESSIVELY
+	select DFU_TARGET
+	select DOWNLOAD_CLIENT
+	select REBOOT
+	select CJSON_LIB
+	select SETTINGS
+	default y
+
+if NRF_CLOUD_COAP_FOTA
+
+config COAP_FOTA_DL_TIMEOUT_MIN
+	int "The time (minutes) allotted for a FOTA download to complete"
+	default 15
+	help
+	  If the FOTA download does not complete in the allotted time,
+	  the download will be cancelled and the job status will be
+	  set as failed.
+
+config COAP_FOTA_USE_NRF_CLOUD_SETTINGS_AREA
+	bool "Use the same settings area as the nRF Cloud FOTA library"
+	default y
+	help
+	  Using the same settings area as the nRF Cloud FOTA library will
+	  allow this sample to perform application-FOTA updates to applications
+	  built with CONFIG_NRF_CLOUD_FOTA enabled.
+
+config COAP_FOTA_SETTINGS_NAME
+	depends on !COAP_FOTA_USE_NRF_CLOUD_SETTINGS_AREA
+	string "Settings identifier for the COAP FOTA sample"
+	default "coap_fota_sample"
+	help
+	  The root settings identifier for this application.
+	  For application-FOTA, this value should also be found in your target application
+	  so that the FOTA status can be properly reported to nRF Cloud.
+
+config COAP_FOTA_SETTINGS_KEY_PENDING_JOB
+	depends on !COAP_FOTA_USE_NRF_CLOUD_SETTINGS_AREA
+	string "Settings item key for pending FOTA job info"
+	default "pending_job"
+	help
+	  The settings item key where pending FOTA job info will be stored.
+	  For application-FOTA, this value should also be found in your target application
+	  so that the FOTA status can be properly reported to nRF Cloud.
+
+config COAP_FOTA_JOB_CHECK_RATE_MINUTES
+	int "FOTA job check interval (minutes)"
+	default 60
+	help
+	  Sets the time to wait between each request to get any pending FOTA job.
+
+config COAP_FOTA_THREAD_STACK_SIZE
+	int "CoAP FOTA Thread Stack Size (bytes)"
+	default 2048
+	help
+	  Sets the stack size (in bytes) for the CoAP FOTA thread of the
+	  sample.
```

Comment:
These are all here temporarily until we integrate this into nrf_cloud itself, correct?

Reply:
Probably not the last two (job check rate minutes and thread stack size), but the rest, yes.